scripts/build-index.js

```diff
@@ -11,48 +11,59 @@ function parsePagename(pagefile) {
 }
 
 function parseLanguage(pagefile) {
-  var pagesFolder = pagefile.split(/\//)[0];
-  return pagesFolder == 'pages' ? 'en' : pagesFolder.replace(/^pages./, '');
+  let pagesFolder = pagefile.split(/\//)[0];
+  return pagesFolder == 'pages' ? 'en' : pagesFolder.replace(/^pages\./, '');
 }
 
 function buildPagesIndex(files) {
-  var reducer = function (index, file) {
-    var os = parsePlatform(file);
-    var page = parsePagename(file);
-    var language = parseLanguage(file);
+  let reducer = function (index, file) {
+    let os       = parsePlatform(file),
```

Comment:
Can we just have single declarations per line like before ? It just seems odd to introduce this spacing convention when it is not used anywhere else in the code.